Formula/go.rb

```diff
@@ -47,11 +37,11 @@ class Go < Formula
   depends_on :macos => :mountain_lion
 
   # Should use the last stable binary release to bootstrap.
-  # More explicitly, leave this at 1.7 when 1.7.1 is released.
+  # More explicitly, leave this at 1.8 when 1.8.1 is released.
```

Comment:
Unless 1.7 has known issues bootstrapping 1.8 I'd probably leave this how it was. The original intention here was that no version of `go` should be bootstrapped by the same version of `go`, so that if there's a problem bootstrapping you know it's with the new from-source release rather than having to work out whether the problem is the binary or the from-source part.